components/massmint/uploadDescription/UploadDescription.vue

```diff
@@ -52,20 +52,34 @@ const acceptedMediaFormatsString = '.TXT, .CSV, .JSON'
 
 const emit = defineEmits(['fileLoaded'])
 
-const onFileSelected = (file: File) => {
+const onFileSelected = (file) => {
   showCheckmark.value = false
-  loading.value = true
 
   // replace true with check for valid file type
-  const { entries, error, loading: isLoading } = useParseDescriptionFile(file)
+  if (true) {
```

Comment:
☁️

Reply:
something is wrong...
i can't get it to properly show the diff with main
changes that came from this PR:
* https://github.com/kodadot/nft-gallery/pull/5824#
are "invisble"